extension/pinch-zoom.js

```diff
@@ -20,7 +20,10 @@ Feel free to get in touch via email if you have any questions
 const isMac = navigator.platform.toLowerCase().indexOf('mac') >= 0;
 const pinchZoomSpeed = isMac ? 0.015 : 0.03;
 const scaleMode = 1; // 0 = always high quality, 1 = low-quality while zooming
-const shiftKeyZoom = true;// enable zoom with shift + scroll
+var shiftKeyZoom = true; // enable zoom with shift + scroll by default
+browser.storage.sync.get('mtzoom_shiftkey', function (res) {
+	shiftKeyZoom = res.mtzoom_shiftkey || true;
```

Comment:
So if `res.mtzoom_shiftkey` is `false` it will be `false || true` whcich is `true`.
```suggestion
	if (typeof res.mtzoom_shiftkey !== 'undefined') {
		shiftKeyZoom = res.mtzoom_shiftkey;
	}
```

Reply:
Good spot @chamini2, thanks :)!

Reply:
That fixes the issue I was having. With this change included it's good to merge

@AkdM, I can't apply the change myself (but I don't have permissions to edit your branch), would you mind applying the change?

Reply:
Oops that is right! I didn't see @haxiomic's comment.